flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/async/AsyncIOExample.java

```diff
@@ -179,7 +179,7 @@ public void close() throws Exception {
 
 		@Override
 		public void asyncInvoke(final Integer input, final ResultFuture<String> resultFuture) throws Exception {
-			this.executorService.submit(new Runnable() {
+			executorService.submit(new Runnable() {
```

Comment:
Did you mean to write "ExecutorService" - with the capital 'E'?

Reply:
no, the `executorService` is static

Reply:
see 10.2 in "Java Code Conventions" (http://www.oracle.com/technetwork/java/codeconventions-150003.pdf)

Reply:
@nekrassov, right. 
These two variables are private(`random`,`executorService`), so can remove static define. do you think so?

Reply:
Sorry, my earlier comment about capital 'E' was wrong - I incorrectly assumed that submit() is a static method

Reply:
don't feel sorry, you are welcome.